### ISSUE TITLE
Polyaxon can't get plxlogs for pytorchjob in dashboard

### ISSUE THREAD

**Issue by hongqing1986** [polyaxon/polyaxon#1505]
###
When i run the pytorchjob, i can't get the plxlogs in dashboard after the job finished,

But if i clicked the logs button of this job in dashboard before the job finished, I can collect the plxlogs.

If i run the common job, the plxlogs is normal.

### yaml config
version: 1
kind: component
tags: [examples, pytorch, kubeflow]
run:
  kind: pytorchjob
  master:
    replicas: 1
    init:
    - git: {"url": "https://github.com/polyaxon/polyaxon-examples"}
    container:
      image: pytorch/pytorch:1.0-cuda10.0-cudnn7-runtime
      command: ["sh", "-c", "python -u {{ globals.artifacts_path }}/polyaxon-examples/in_cluster/kubeflow/pytorchjob/mnist.py"]
      resources:
        requests:
          nvidia.com/gpu: 1
  worker:
    replicas: 1
    init:
    - git: {"url": "https://github.com/polyaxon/polyaxon-examples"}
    container:
      image: pytorch/pytorch:1.0-cuda10.0-cudnn7-runtime
      command: ["sh", "-c", "python -u {{ globals.artifacts_path }}/polyaxon-examples/in_cluster/kubeflow/pytorchjob/mnist.py"]
      resources:
        requests:
          nvidia.com/gpu: 1

**Comment by hongqing1986** [comment]
Current cli version: 1.18.0


Platform version:

                                          
 KEY     182a99f9ce4853e38cc57644500fd73a 
 VERSION 1.18.1                           
 DIST    ce                               
                                          

Compatibility versions:

                                                
 cli      {"min": "1.10.0", "latest": "1.18.2"} 
 platform {"min": "1.10.0", "latest": "1.18.2"} 
 agent    {"min": "1.10.0", "latest": "1.18.2"} 
 ui       {"min": "1.10.0", "latest": "1.18.2"} 
                                                
New version of Polyaxon CLI (1.18.2) is now available. To upgrade run:
    pip install -U polyaxon

**Comment by polyaxon-team** [comment]
We will try to reproduce the problem and report back.

**Comment by hongqing1986** [comment]
-- Also please report what verion of the training-operator you are using.
How can i get the training-operator?
I just use helm to install the trainingjobs.

**Comment by polyaxon-team** [comment]
Yes, I assumed you are using the latest version of the [polyaxon/trainingjobs](https://polyaxon.com/integrations/kubeflow/#deployingdeleting-the-trainingjobs-operator) `1.15.1`, you can check that using:

```bash
helm search repo polyaxon
```

**Comment by hongqing1986** [comment]
Where is the polyaxon-operator's source code address?
Is it also in https://github.com/polyaxon/polyaxon?

**Comment by hongqing1986** [comment]
Is there any feedback?

**Comment by hongqing1986** [comment]
@polyaxon-team Have you reproduce the problem?

**Comment by hongqing1986** [comment]
Is there any feedback?

**Comment by hongqing1986** [comment]
I have configured the artifactsStore, which is a nfs directory.
`namespace: polyaxon
rbac:
  enabled: true
postgresql:
  persistence:
    enabled: true
    existingClaim: polyaxon-pg-pvc
artifactsStore:
  name: artifacts-store
  kind: volume_claim
  schema:
    mountPath: "/artifacts-store"
    volumeClaim: "polyaxon-artifacts-pvc"
intervals:
  compatibilityCheck: -1
operators:
  tfjob: true
  pytorchjob: true
  mpijob: true`

**Comment by hongqing1986** [comment]
Following is the polyaxon's pods, i don't know if I missed some pods.
NAME    READY   STATUS
polyaxon-polyaxon-api-7476497d9f-zr6xl  1/1     Running
polyaxon-polyaxon-gateway-7fb978d658-rss9m      1/1     Running
polyaxon-polyaxon-operator-8987bb494-hzxsh      1/1     Running
polyaxon-polyaxon-streams-5bff76b98d-hlqww      1/1     Running
polyaxon-postgresql-0   1/1     Running
trainingjobs-trainingjobs-7989c46878-s46wd      1/1     Running
